src/functions/meetup.js

```diff
@@ -168,11 +192,16 @@ exports.handler = async (event, context, callback) => {
     }
 
     // create
-    // console.log(`Creating entry ${meetup.eventName}`)
+    console.log(`Creating entry ${meetup.eventName}`)
     const id = await environment.createEntry('meetupEven', entry)
     const newEntry = await environment.getEntry(id.sys.id)
 
-    // console.log(`Publishing creted entry ${meetup.eventName}`)
+    console.log(`Publishing creted entry ${meetup.eventName}`)
     return newEntry.publish()
   })
+
+  return {
+    statusCode: 200,
+    body: 'Meetup function has finished running'
```

Comment:
maybe its worth including what events were published to contenful here? It'll  be logged in the AWS lambda console and be helpful for debugging

Reply:
ah ignore me this will be logged in the console logs above